pyhf/utils.py

```diff
@@ -0,0 +1,27 @@
+import json, jsonschema
+import pkg_resources
+
+from .exceptions import InvalidSpecification
+
+DEFAULT_SCHEMA = pkg_resources.resource_filename(__name__,'data/spec.json')
+def get_default_schema():
+  global DEFAULT_SCHEMA
+  return DEFAULT_SCHEMA
+
```

Comment:
why can this not simply be

```
def get_default_schema():
    return pkg_resources.resource_filename(__name__,'data/spec.json')
```

Reply:
Yeah, probably can change to just that. I wasn't sure about the repetitive function calls to `pkg_resources` etc... but it's a silly optimization here.